### PR TITLE
Implemented art feature

### PR DESCRIPTION
**Pull request by PayalHake1402**
issue : Create art creator feature #116

Art_Feature_UI

So basically I have to write CSS on the left side, and those CSS rules will render in the right side. When anyone visit our sight and try to create something art so for that we have to implement this feature .

### CODE REVIEW

**Comment by PayalHake1402** [comment]
I have created a pull request with updated changes . Please review it.

File: art-feature-august-2021.html [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+        if ((str === null) || (str === ''))
+          return false;
+        
+        str = str.toString();
+
+        return str.replace(/(<([^>]+)>)/ig, '');
+      }
+
+
+      function run() {
```
Comment (harshith-venkatesh) [inline art-feature-august-2021.html:41]: please follow indentation as the code is not properly indented

File: art-feature-august-2021.html [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+          .setAttribute("html-code", sanitizedHtmlCode);
+
+        const output = document.querySelector("#output");
+        output.contentDocument.body.innerHTML = sanitizedHtmlCode;
+
+      }
+
+      document
+        .querySelector("#html-code")
+        .addEventListener("keyup", run);
```
Comment (harshith-venkatesh) [inline art-feature-august-2021.html:66]: can we have more meaningful function name please

File: css/main.css [hunk 3]
```diff
@@ -311,3 +311,66 @@ footer {
...
+
+.welcome-title {
+  margin-top: 20px;
+  font-size: 2.75rem;
+  text-align: center;
+}
+.editor {
+  margin-top: 70px;
+  display: flex;
+  flex-direction: row;
```
Comment (harshith-venkatesh) [inline css/main.css:325]: flex direction is row by default, we can remove this

**Review by harshith-venkatesh — CHANGES_REQUESTED** [review]

File: art-feature-august-2021.html (hunk 2 above)
Comment (harshith-venkatesh) [reply art-feature-august-2021.html:66]: Probably, like sanitizedCodeOutput , sanitizedHtml
Just few suggestions, looks more descriptive too right

**Comment by rohan09-raj** [comment]
There were no further developments That's why this has been already taken care of and merged with PR #155

**Pull request closed without merging**